packages/nuxt/src/app/nuxt.ts

```diff
@@ -142,7 +142,7 @@ export const NuxtPluginIndicator = '__nuxt_plugin'
 
 export interface PluginMeta {
   name?: string
-  enforce?: 'pre' | 'post'
+  enforce?: 'payload' | 'pre' | 'post'
```

Comment:
`payload` as `enforce` can be kinda confusing because we might as well also have a post-post enforce that is to modify final `payload` while this (i suppose) is for payload reducers.

Would be nice to make it more clear

Reply:
Can you think of a better name? Maybe `payload-type` or `custom-type`?

Reply:
What if we switch to a numeric order? We can map `pre` and `post` for convenience to user values but this way we have full control and can add nuxt internal pre(s) before/after user pre(s). Something like this

- -10: nuxt-pre-all
- -5: user-pre
- -1: nuxt-default
- 0: default
- 1: nuxt-after
- +5: user-after
- +10: nuxt-after-all

Reply:
Perfect. 👌

Reply:
Re payloads, we could also make a special `definePayloadPlugin` wrapper what do you think?